Mathlib/Algebra/Star/Order.lean

```diff
@@ -34,6 +34,8 @@ positive cone which is the _closure_ of the sums of elements `star r * r`. A wea
 the advantage of not requiring a topology.
 -/
 
+open Set
+
```

Comment:
```suggestion
```
This seems to do nothing since there is no other change in that file.

Reply:
Whoops sorry yes. I added changes and subsequently deleted them.